spec/lib/external_count_fetcher_spec.rb

```diff
@@ -33,7 +33,7 @@ def counts_hash(count: )
         expect(count_fetcher.house_supporters.ttl).to eq(ExternalCountFetcher::REDIS_EXPIRE_SECONDS)
       end
       Timecop.freeze do
-        new_time = (ExternalCountFetcher::REDIS_EXPIRE_SECONDS-20.minutes.to_i)
+        new_time = (ExternalCountFetcher::REDIS_EXPIRE_SECONDS/3)
```

Comment:
This test was failing before. This seems to fix it. I'm not really sure what this testing, though. If I'm reading it right, we're going back in time to set a Redis value and then coming forward in time to make sure that Redis correctly manages expiration. Isn't that outside the scope of what we need to test?

P.S. we can now retire Timecop if we want to: https://blog.pivotal.io/labs/labs/goodbye-timecop-hello-timehelpers